create_account/forms.py

```diff
@@ -0,0 +1,32 @@
+from django import forms
+from django.contrib.auth.models import User
+from django.contrib.auth.forms import UserCreationForm
+from home import models
+
+class ExtendedUserCreationForm(UserCreationForm):
+    email=forms.EmailField(required=True)
+    first_name=forms.CharField(max_length=50)
+    last_name=forms.CharField(max_length=50)
+
+    class Meta:
+        model=User
+        fields=('username','email', 'first_name', 'last_name')
```

Comment:
@akshay4241 I was implementing the same thing for creating employee accounts but you may want to look at Chandana's  version of this file. It's a bit different. We may have to make changes accordingly so that it blends with her front end code.